applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -175,6 +175,14 @@ static void stream_recv_cb(struct bt_audio_stream *stream, const struct bt_iso_r
 		bad_frame = true;
 	}
 
+	if (buf->len != active_stream.codec->octets_per_sdu) {
+		data_size_mismatch_cnt++;
+		if ((data_size_mismatch_cnt % 500) == 1) {
+			LOG_WRN("Data size mismatch, received: %d, codec: %d, total: %d", buf->len,
```

Comment:
Should be a LOG_DBG. Zero size packets are sent from time to time if there is no data to be sent on the gateway side. This goes for all three of them

Reply:
Changed to LOG_DBG